ui/src/components/overview/MobileClientCardViewItem.js

```diff
@@ -30,42 +31,47 @@ const MobileClientCardViewItem = props => {
-        <CardHeading>
-          <DropdownKebab id={app.metadata.name} pullRight className="card-dropdown-kebab">
-            <EditItemButton item={app} />
-            <DeleteItemButton itemType="app" itemName={appName} item={app} />
-          </DropdownKebab>
-          <CardTitle>
-            {buildTabEnabled && builds.numFailedBuilds > 0 ? (
-              <span>
-                <span className="pficon pficon-error-circle-o" />
-                {builds.numFailedBuilds}
-              </span>
-            ) : null}
-            {buildTabEnabled && builds.numInProgressBuilds > 0 ? (
-              <span>
-                <span className="pficon fa fa-refresh fa-spin fa-fw" />
-                {builds.numInProgressBuilds}
-              </span>
-            ) : null}
-            {!buildTabEnabled || (builds.numFailedBuilds === 0 && builds.numInProgressBuilds === 0) ? <span /> : null}
-          </CardTitle>
-        </CardHeading>
+      <Link to={`/mobileclient/${app.metadata.name}`}>
+        <Card matchHeight /* accented */ className="mobile-client-card">
+          <CardHeading>
+            <DropdownKebab id={app.metadata.name} pullRight className="card-dropdown-kebab">
+              <EditItemButton item={app} />
+              <DeleteItemButton itemType="app" itemName={appName} item={app} />
+            </DropdownKebab>
+            <div className="card-pf-title">
+              <h1>{app.spec.name}</h1>
+            </div>
+
+            <CardTitle>
+              {buildTabEnabled && builds.numFailedBuilds > 0 ? (
+                <span>
+                  <span className="pficon pficon-error-circle-o" />
+                  {builds.numFailedBuilds}
+                </span>
+              ) : null}
+              {buildTabEnabled && builds.numInProgressBuilds > 0 ? (
+                <span>
+                  <span className="pficon fa fa-refresh fa-spin fa-fw" />
+                  {builds.numInProgressBuilds}
+                </span>
+              ) : null}
+              {!buildTabEnabled || (builds.numFailedBuilds === 0 && builds.numInProgressBuilds === 0) ? <span /> : null}
+            </CardTitle>
+          </CardHeading>
 
-        <Link to={`/mobileclient/${app.metadata.name}`}>
           <CardBody>
-            <div className="card-body-title">
-              <h1>{app.spec.name}</h1>
+            <div className="card-icons">
+              {services && services.length > 0 ? getServiceIcons(services) : <div className="service-icon" />}
             </div>
           </CardBody>
           <CardFooter>
-            <div className="card-footer-icons">
-              {services && services.length > 0 ? getServiceIcons(services) : <div className="service-icon" />}
-            </div>
+            <div className="creation-timestamp">CREATED</div>
+            <span className="creation-timestamp">
+              <span className="fa fa-globe" /> <Moment format="DD/MM/YYYY">{app.metadata.creationTimestamp}</Moment>
```

Comment:
Shouldn't we match locale of browser/serverside for date formatting? Or are we using US weird date format :)

Reply:
Not sure about that: none of our messages is localized, so I don't think we should localize the dates. WDYT?

Reply:
yeah, let's not worry about localisation yet.  If we are going to do it, it will be part of the work to  support i18n in the UI.

Reply:
ok for me